includes/degrees.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * Commands for converting degrees
+ */
+class Degrees extends WP_CLI_Command {
+    /**
+     * Converts Main-Site-Themes degrees to v3 degrees.
+     *
+     * ## EXAMPLES
+     * 
+     * $ wp mainsite degrees convert
+     *
+     * @when after_wp_load
+     */
+    public function convert( $args ) {
+        $query_args = array(
+            'post_type'    => 'degree',
+            'numberposts'  => -1,
+            'post_status'  => 'any'
+        );
+
+        $posts = get_posts( $query_args );
+
+        foreach( $posts as $post ) {
+            $this->convert_post_meta( $post );
+        }
+
+        WP_CLI::success('Successfully converted degrees...');
+    }
+
+    /**
+     * Removes all degrees
+     *
+     * ## EXAMPLES
+     * 
+     * # Removes all degrees
+     * $ wp mainsite degrees reset
+     * 
+     * @when after_wp_load
+     */
+    public function reset( $args ) {
```

Comment:
This function needs to be removed when development is done. This is really here as a debugging tool.